1-org/modules/centralized-logging/variables.tf

```diff
@@ -25,15 +25,27 @@ variable "resources" {
 }
 
 variable "resource_type" {
-  description = "Resource type of the resource that will export logs to destination. Must be: project, organization, or folder."
+  description = "Resource type of the resource that will export logs to destination. Must be: project, organization or folder."
   type        = string
 
   validation {
     condition     = contains(["project", "folder", "organization"], var.resource_type)
-    error_message = "The resource_type value must be: project, organization, or folder."
+    error_message = "The resource_type value must be: project, organization or folder."
   }
 }
 
+variable "billing_account" {
+  description = "Billing Account ID used in case sinks are under billing account level. Format 000000-000000-000000."
+  type        = string
+  default     = null
+}
+
+variable "enable_billing_account_sink" {
```

Comment:
Since we are declaring the `enable_billing_account_sink` input variable for `centralized-logging` module, I believe it would make sense to pass it as an argument during the `centralized-logging` module call (in `1-org/envs/shared/log_sinks.tf)`, no?

The way it is right now, because the default is `false` and we are not passing this input in the module's call, we will never have the sink creation for the billing account.